### DISCUSSION TITLE
Move asdf-kubectl here

### DISCUSSION THREAD

**Discussion by DustinChaloupka** [asdf-community/.github#17] [General]
Would like to move https://github.com/Banno/asdf-kubectl to the community org. Thanks!

**Comment by smorimoto** [comment]
Please do it. Let it transfer here!

**Reply by DustinChaloupka** [reply]
It has been transfered.

**Reply by smorimoto** [reply]
Done in https://github.com/asdf-community/infrastructure/commit/1ef00ee2bdaf6179987ff2ecdd380b94643abab2.
